src/ui/server/controllers/insurance/policy-and-export/map-submitted-data/policy-contact/index.test.ts

```diff
@@ -112,4 +100,32 @@ describe('controllers/insurance/policy-and-export/map-submitted-data/policy-cont
       expect(result).toEqual(expectedVariables);
     });
   });
+
+  describe(`when ${OTHER_NAME} the same as owner`, () => {
+    const mockBody = {
+      [NAME]: OTHER_NAME,
+      ...mockContact,
+    };
+
+    const applicationSameOwner = mockApplication;
+
+    beforeEach(() => {
+      applicationSameOwner.owner = mockContact;
+    });
```

Comment:
Maybe this could go into the `it('...` like this to save some lines 

```js
const applicationSameOwner = {
  ...mockApplication,
  owner: mockContact,
};
```